pytest/backend/test_states_polyquad.py

```diff
@@ -23,6 +23,8 @@
 from strawberryfields import utils
 from strawberryfields.backends.shared_ops import rotation_matrix as R, changebasis
 
+# some tests require a higher cutoff for accuracy
+CUTOFF = 12
```

Comment:
Do we want to have the test explicitly print a warning/msg that the user-supplied/default cutoff is ignored for this test?